src/Microsoft.ML.FastTree/TreeTrainersCatalog.cs

```diff
@@ -107,7 +107,7 @@ public static class TreeExtensions
         /// <param name="minDatapointsInLeaves">The minimal number of datapoints allowed in a leaf of the tree, out of the subsampled data.</param>
         /// <param name="learningRate">The learning rate.</param>
         /// <param name="advancedSettings">Algorithm advanced settings.</param>
-        public static BinaryClassificationGamTrainer GeneralizedAdditiveMethods(this BinaryClassificationContext.BinaryClassificationTrainers ctx,
+        public static BinaryClassificationGamTrainer GeneralizedAdditiveModels(this BinaryClassificationContext.BinaryClassificationTrainers ctx,
```

Comment:
>GeneralizedAdditiveModels [](start = 53, length = 25)

same comment as the other PR, samples :)

Reply:
Definitely!